{Release} Upgrade to Azure CLI 2.86.0 (#33275)
Azure/azure-cli: src/azure-cli/HISTORY.rst

* `az aks mesh enable/proxy-redirection-mechanism`: Add mesh Istio CNI commands (#32992)
* `az aks nodepool add`: Add `Windows2025` as a supported value for `--os-sku` (#33246)
* `az aks update`: Fix `--enable-azure-monitor-metrics` failing with RoleAssignmentExists when Grafana role assignment already exists (#33229)

**App Service**

* `az webapp create`: Add post-creation deployment guidance and startup command examples (#33088)
* `az webapp deploy`: Add build automation guidance for Linux zip deployments (#33088)
* `az webapp up`: Add logging for OS type, runtime auto-detection, and plan auto-generation (#33088)
* Add warning for container image path when `--container-image-name` includes registry host (#33088)
* `az webapp delete`: Add warning about App Service Plan deletion (#33088)
* Fix #33180: `az functionapp plan create`: Simplify reserved parameter assignment in AppServicePlan (#33202)
* `az webapp sitecontainers convert`: Add support for converting Docker Compose multi-container apps to Sitecontainers mode (#33131)
* `az webapp up/deploy`: Add `--enriched-errors` parameter to see detailed deployment failure log (#32940)

> * `az webapp create`: Add error message that clearly lists all valid options and tells them how to discover available runtimes (#33252)

Reviewer: ```suggestion
* `az webapp create`: Add error message that clearly lists all valid options and specifies how to discover available runtimes (#33252)
```

* `az appservice plan create`: Make `P0V3` as default SKU when `--sku` is omitted for linux webapp (#33237)
* `az appservice plan create`: Add `PREMIUM0V3` tier for elastic scale (#33237)

**Cloud**

> * known clouds list / AZURE_BLEU_CLOUD : changed active_directory_endpoint : "login.sovcloud-api.fr" -> "login.sovcloud-identity.fr" (#33182)

Reviewer 2: ```suggestion
* Fix #33183: `az cloud set`: Typo correction on AZURE_BLEU_CLOUD active directory endpoint
```

**Compute**

* `az sig create`: Add new argument group "Managed Service Identity" to configure the service identity of a Shared Image Gallery (SIG) (#33137)
* `az sig show`: Update command to display the managed service identity of a Shared Image Gallery (SIG) (#33137)
* `az sig identity`: Add new command group to manage the service identity of a Shared Image Gallery (SIG) (#33137)

**Container app**

* `az containerapp create`: Support other cloud for acr (#33160)

**Network**

> * Add Az Cli commands for Vwan gateways Resiliency APIs (#32941)

Reviewer 3: ```suggestion
* `az network express-route gateway`: Support VWAN gateway resiliency APIs (#32941)
```

* `az network route-table create/update`: Add `--disable-peering-route` to support disable peering route (#33231)

**PostgreSQL**

* `az postgres flexible-server create`, `replica create`, `restore`, `geo-restore`, and `revive-dropped`: Add breaking change announcement for command behavioral change related to network resources (#33077)
* `az postgres flexible-server create`: Announce breaking change deprecation of `--cluster-option` argument (#33225)
* Fix #33090: `az postgres flexible-server server-logs list`: Command crashes with an AttributeError when listing log files (#33096)
* `az postgres flexible-server`: Improve validation logic and style of error messages (#33114)
* `az postgresql flexible-server replica create`: New argument `--storage-type` to select storage type as PremiumV2_LRS for read replica (#33134)
* Fix #33205: `az postgres flexible-server create`: Update help text for `--storage-auto-grow` argument to reflect actual default value (#33206)
* `az postgres flexible-server update`: Restart is no longer required for scaling storage size of Premium SSDv2 server (#33178)

> * `az postgres flexible-server create/upgrade`: Block SSDv2 creation for PG version < 14 (#33119)

Reviewer: ```suggestion
* `az postgres flexible-server create/upgrade`: Block SSDv2 creation for PG version earlier than 14 (#33119)
```
